pytorch_lightning/accelerators/accelerator.py

```diff
@@ -228,7 +228,7 @@ def validation_step_end(self, output: Optional[STEP_OUTPUT]) -> Optional[STEP_OU
         """
         return self.training_type_plugin.validation_step_end(output)
 
-    def backward(self, closure_loss: Tensor, *args: Any, **kwargs: Any) -> Tensor:
+    def backward(self, closure_loss: Tensor, optimizer: torch.optim.Optimizer, *args: Any, **kwargs: Any) -> Tensor:
```

Comment:
as a side note: this new parameter would be removed again with the long-term solution of moving the `step` into the precision plugin